sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/rdf/RdfSubject.java

```diff
@@ -37,6 +39,24 @@ default TriplePattern has(RdfPredicate predicate, RdfObject... objects) {
 		return GraphPatterns.tp(this, predicate, objects);
 	}
 	
+	/**
+	 * Create a triple pattern from this subject and the given predicate and
+	 * object
+	 * 
+	 * @param predicate
+	 *            the predicate {@link IRI} of the triple pattern
+	 * @param objects
+	 *            the object(s) of the triple pattern
+	 * @return a new {@link TriplePattern} with this subject, and the given predicate and object(s)
+	 * 
+	 * @see <a
+	 *      href="http://www.w3.org/TR/2013/REC-sparql11-query-20130321/#QSynTriples">
+	 *      Triple pattern syntax</a>
+	 */
+	default TriplePattern has(IRI predicate, RdfObject... objects) {
+		return has(Rdf.iri(predicate), objects);
+	}
+	
```

Comment:
I think this should include a couple methods that take `Value...` as well, no? One for `RdfPredicate` and `Value...` and one for `IRI` and `Value...`?

Reply:
Yeah, missed that one!